docs/index.md

```diff
@@ -50,13 +50,16 @@ Components:
     - [Alert CRD](components/notification/alert.md)
     - [Receiver CRD](components/notification/receiver.md)
 
-To get started with the toolkit please follow this [guide](get-started/index.md).
+## Get Started
+
+!!!hint "Get started with the GitOps Toolkit!"
```

Comment:
Given this already renders a title, do we need the `# Get Started`?

Reply:
![Screenshot from 2020-10-06 14-43-41](https://user-images.githubusercontent.com/1346979/95202989-609f7b80-07e2-11eb-83f7-60a6772bf21f.png)

Reply:
We could point to some individual guides in that section further down the line?

Reply:
Oh, this looks great as is.